hyperjumptech/monika
# Feat: add yaml file config support

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
Need to add Yaml support for improving monika configuration.

## How did you implement / how did you fix it  
1.  add check for default monika.yaml or monika.yml file if monika.json or config.json is not found
2. check config file extension when parsing 
3. parse yaml configuration file and convert it to Config object 

## How to test  
1. copy monika.example.yml to monika.yml
2. run `npm start`
3. monika should run successfully
